main.go

```diff
@@ -36,10 +36,16 @@ var (
 	clusterCmd = app.Command("cluster", "Do some action on cluster(s).")
 
 	domainCmd     = app.Command("domain", "Do some action on domain(s).")
-	domainCluster = domainCmd.Flag("cluster", "Cluster ID.").Short('c').String()
+	domainCluster = domainCmd.Flag("cluster", "Cluster ID. This flag requires a domain ID, domain name will not work.").Short('c').String()
```

Comment:
Confusing wording. What you mean to say is: "When this option is given, the domain must be identified by ID. Specifiying a domain name will not work." (Same for `var projectCluster` below.)